### PR TITLE
Feature/ob dex

### PR DESCRIPTION
**Pull request by Chepelau**

### CODE REVIEW

**Comment by lukaspetrasek** [comment]
Can you pls rebase so that the pipelines are triggered?

File: Dockerfile.ob-liq-fetching [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+RUN pip install poetry
+COPY pyproject.toml poetry.lock* ./
+RUN poetry config virtualenvs.create false \
+  && poetry install --no-interaction --no-ansi
+
+COPY ./scripts ./scripts
+COPY ./src ./src
+COPY ./db ./db
+
+RUN python3 -m pip install anchorpy
```
Comment (lukaspetrasek) [inline Dockerfile.ob-liq-fetching:15]: Pls move this block above the `COPY` block.

File: Dockerfile.ob-liq-fetching [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  && poetry install --no-interaction --no-ansi
+
+COPY ./scripts ./scripts
+COPY ./src ./src
+COPY ./db ./db
+
+RUN python3 -m pip install anchorpy 
+RUN python3 -m pip install solana==0.28.1
+RUN python3 -m pip install mango_explorer_v4
+RUN python3 -m pip install typing-extensions --upgrade
```
[the remark this answers is not in the document]
Comment (Chepelau) [reply Dockerfile.ob-liq-fetching:18]: Poetry just doesn't want to add some of them so yep. (funny thing, sqlalchemy stops working after install mango_explorer_v4 if I don't run the typing-extensions upgrade)

**Comment by Chepelau** [comment]
Rebased. Added sqlalchemy mypy plugin.

**Pull request merged**